communities/massmesh/meshnode/files/etc/config/system

```diff
@@ -0,0 +1,19 @@
+config system
+    option hostname 'OpenWrt'
+    option ttylogin '1'
+    option log_size '64'
+    option urandom_seed '1'
+    option zonename 'America/New York'
+    option timezone 'EST5EDT,M3.2.0,M11.1.0'
+    option log_proto 'udp'
+    option conloglevel '8'
+    option cronloglevel '8'
+
+
+config timeserver 'ntp'
+    option enabled '1'
```

Comment:
supply timeserver, but unknown if dhcp/dnsmasq supplies these options. some devices must absolutely have time or they will not boot (apple tv and other) and/or https will not work.